ext/date/tests/DateTime_wakeup_exception.phpt

```diff
@@ -0,0 +1,34 @@
+--TEST--
+DateTime invalid serialization data (wakeup)
+--FILE--
+<?php
+$propertySets = [
+	'O:8:"DateTime":3:{s:4:"date";s:26:"2023-01-13 14:48:01.705516";s:13:"timezone_type";i:3;s:8:"timezone";s:13:"Europe/London";}',
+	'O:8:"DateTime":3:{s:4:"date";i:3;s:13:"timezone_type";i:3;s:8:"timezone";s:3:"UTC";}',
+	'O:8:"DateTime":3:{s:4:"date";s:26:"2023-01-13 14:48:01.705516";s:13:"timezone_type";s:1:"X";s:8:"timezone";s:3:"UTC";}',
+	'O:8:"DateTime":3:{s:4:"date";s:26:"2023-01-13 14:48:01.705516";s:13:"timezone_type";i:4;s:8:"timezone";s:3:"UTC";}',
+	'O:8:"DateTime":3:{s:4:"date";s:26:"2023-01-13 14:48:01.705516";s:13:"timezone_type";i:3;s:8:"timezone";s:11:"Europe/Lviv";}',
+	'O:8:"DateTime":3:{s:4:"date";s:26:"2023-01-13 14:48:01.705516";s:13:"timezone_type";i:3;s:8:"timezone";s:11:"Europe/Kyiv";}',
+	'O:8:"DateTime":3:{s:4:"date";s:26:"2023-01-13 14:48:01.705516";s:13:"timezone_type";i:3;s:8:"timezone";s:1:"Europe/Kyiv";}',
+];
+
+foreach( $propertySets as $propertySet )
+{
+	try {
+		$d = unserialize($propertySet);
+		echo "OK? ", gettype($d), "\n";
+	} catch (\Error $e) {
+		echo $e->getMessage(), "\n";
+	}
+}
+?>
+--EXPECTF--
+OK? object
```

Comment:
Just to make sure: The first is actually valid data? This contradicts the test name / description then.